documentation/docs/features/smoke/index.md

```
@@ -14,6 +14,7 @@
 Возможности:
 
 * [Тестирование открытия форм](opening-forms.md)
+* [Запись справочников]()
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)